packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -0,0 +1,59 @@
+import chalk from 'chalk';
+
+import {
+  StatuspageServiceFragment,
+  StatuspageServiceName,
+  StatuspageServiceStatus,
+} from '../graphql/generated';
+import { StatuspageServiceQuery } from '../graphql/queries/StatuspageServiceQuery';
+import Log, { link } from '../log';
+
+export async function maybeWarnAboutEasOutagesAsync(
+  serviceNames: StatuspageServiceName[]
+): Promise<void> {
+  const services = await getStatuspageServiceAsync(serviceNames);
+
+  for (const service of services) {
+    warnAboutServiceOutage(service);
+  }
+}
+
+const humanReadableServiceName: Record<StatuspageServiceName, string> = {
+  [StatuspageServiceName.EasBuild]: 'EAS Build',
+  [StatuspageServiceName.EasSubmit]: 'EAS Submit',
+  [StatuspageServiceName.EasUpdate]: 'EAS Update',
+};
+
+function warnAboutServiceOutage(service: StatuspageServiceFragment): void {
+  if (service.status !== StatuspageServiceStatus.Operational) {
```

Comment:
Invert to get rid of one level of indentation.
```suggestion
  if (service.status === StatuspageServiceStatus.Operational) {
    return;
  }
```